src/js/containers/covid19/recipient/RecipientContainer.jsx

```diff
@@ -18,13 +20,27 @@ const RecipientContainer = () => {
         <div className="body__content recipient__container">
             <DateNote />
             <h3 className="body__narrative">
-                These are the recipients who received COVID-19 Response <strong>awards</strong>.
+                <strong>Who</strong> received funding through COVID-19 Response awards?
             </h3>
-            <p className="body__narrative-description">
-                Federal agencies allocate award funds. Agencies receive funding from the Federal
-                Government, which they award to recipients in order to respond to the COVID-19
-                pandemic.
-            </p>
+            <div className="body__narrative-description">
+                <p>
+                    Once the federal government has determined that an individual, organization, business, or state, local, or tribal government will receive an award, the money is obligated (promised) and then outlayed (paid) according to the terms of the contract or financial assistance.<sup>3</sup>
+                </p>
+                <ReadMore>
+                    <p>
+                        In the map below, you will see a spending breakdown by state, county, or congressional district of recipients who have received awards funded by COVID-19 appropriations. These breakdowns are based on recipients’ addresses. In the case of individual recipients, data is aggregated by county or state to protect personally identifiable information (PII). Data about the location where the award money is used, known as the ‘Primary Place of Performance’, is available in the profile page download.
```

Comment:
Looks like download is strong? 
<img width="811" alt="Screen Shot 2020-07-29 at 2 35 14 PM" src="https://user-images.githubusercontent.com/14301977/88840103-fba82380-d1a9-11ea-8d79-4c64f9dc8160.png">